patches/tModLoader/Terraria/ModLoader/Core/ModCompile.cs

```diff
@@ -54,20 +54,26 @@ public BuildingMod(TmodFile modFile, BuildProperties properties, string path) :
 
 	public static readonly string ModSourcePath = Path.Combine(Program.SavePathShared, "ModSources");
 
-	internal static string[] FindModSources()
+	internal static string[] FindModSources(bool refindAllMods, out LocalMod[] builtMods)
```

Comment:
The extra arguments are just kinda painful. With my other comment we don't really need them.